ccd-definition/CaseEventToFields/createOrder.json

```diff
@@ -25,6 +26,33 @@
     "PageColumnNumber": 1,
     "FieldShowCondition": "orderTypeAndDocument=\"DO_NOT_SHOW\"",
     "ShowSummaryChangeOption": "N"
+  },{
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder",
+    "CaseFieldID": "showFinalOrderSingleChildPage",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "READONLY",
+    "PageID": "OrderTypeAndDocument",
+    "PageLabel": " ",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "orderTypeAndDocument=\"DO_NOT_SHOW\"",
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder",
+    "CaseFieldID": "remainingChildIndex",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "READONLY",
```

Comment:
could update page field display order to 4? (although I know we don't display it on the UI so maybe you wanted to keep as 3?

Reply:
Missed that thanks